### DISCUSSION TITLE
!kc for Lizardman Shamans

### DISCUSSION THREAD

**Discussion by Joepicklebob** [runelite/runelite#15227] [Ideas]
A suggestion was already mentioned like having !kc Shamans work. Allow the option to visit the Shayzien Infiltrator and manually update !kc command since it doesn't track normally. would that work? The same could be said for Demonic Gorilla since they have the grave or collection log to check KC or the slayer monster log for other monsters.

#9581 was the old suggestion but not sure if messaging in a closed discussion will reopen it.

**Comment by Adam-** [comment]
Having !kc for things which are only manually updateable is fairly bad ux since it is unintuitive and dissimilar to how all of the other kc commands work.

**Reply by Joepicklebob** [reply]
It's how the !pets command works, you need to open the Collection log to update the command

**Reply by Alexsuperfly** [reply]
Incorrect, it will automatically track new pets.

**Reply by Joepicklebob** [reply]
Unless that is new i just got a pet 2 months ago and tried !pets and had to open Collection log to update command since i never used it

**Reply by Adam-** [reply]
It tracks the new pets if you have the collection log update message on.

**Reply by Joepicklebob** [reply]
I have it on

**Reply by Joepicklebob** [reply]
Also, the loot tracker tracks KC so it should be able to track a !KC command.
